### ISSUE TITLE
0.33 compat

### ISSUE THREAD

**Issue by rdp** [kemalcr/kemal-session#84]
Today:

```
There was a problem expanding macro 'define_storage'

Code in lib/kemal-session/src/kemal-session/engines/memory.cr:52:9

 52 | define_storage({
      ^
Called macro defined in lib/kemal-session/src/kemal-session/engines/memory.cr:7:9

 7 | macro define_storage(vars)

Which expanded to:

 > 22 |         
 > 23 |           @ints = Hash(String, Int32).new
 > 24 |           @last_access_at = Time.new.to_unix_ms
                                         ^--
Error: no overload matches 'Time.new'
```

**Comment by Destructed3** [comment]
Breaks due to [recent deprecations](https://crystal-lang.org/2020/02/14/crystal-0.33.0-released.html#standard-library).

**Comment by sdogruyol** [comment]
Fixed via #81 :+1:
